cmd/reviewdog/main.go

```diff
@@ -89,10 +89,16 @@ const (
 		"nofilter"
 			Do not filter any results.
 `
-	reporterDoc = `reporter of reviewdog results. (local, github-check, github-pr-check, github-pr-review, gitlab-mr-discussion, gitlab-mr-commit, gitea-pr-review)
+	reporterDoc = `reporter of reviewdog results.
 	"local" (default)
 		Report results to stdout.
 
+	"rjson"
+		Report results to stdout in rdjson format.
+
+	"rjsonl"
```

Comment:
```suggestion
	"rdjsonl"
```

I think the above is correct.